crates/storage/src/entities.rs

```diff
@@ -261,6 +262,7 @@ pub trait AtomicUnit: Data {}
 /// use calimero_storage::entities::{ChildInfo, Data, Element};
 ///
 /// #[derive(AtomicUnit, Clone, Debug, Eq, PartialEq, PartialOrd)]
+/// #[type_id(42)]
```

Comment:
developers have to manage this count themselves?

Reply:
At present yes. As I've indicated in the code, some areas are longhand for now, to establish working functionality, and can be tided into macros later. However, this part will likely never become fully automatic, as there needs to be some independent consistency in order to maintain the same association throughout various code changes. So I think this will be as it currently is, or an approach that is similar enough, i.e. the type ID needs to be statically established somewhere.